sessions/smoother-collaboration.qmd

```diff
@@ -360,16 +416,23 @@ don't need to go over it. Especially mention the second part of the tip.
 
```

Comment:
Not sure what "second part of the tip" the instructor note above refers to? Maybe it should be removed?

Reply:
I'll look at it.